doc/capstone.md

```diff
@@ -4,15 +4,16 @@ Capstone
 Capstone Engine is the disassembler engine used by Rizin by default for 
 some architectures.
 
-Rizin supports capstone 3, 4 and 5.
+Rizin supports capstone 3, 4 and next.
 
 * capstone3: legacy support (only for Debian probably)
 * capstone4: stable release at the moment of writing this
-* capstone5: next branch, still under development
+* capstone-next: next branch, still under development
 
 By default Rizin will build statically against a bundled version of capstone4
-(with some custom patches applied). Please be aware that by default Rizin will
-download Capstone in the source directory `shlr/capstone`.
+(with some custom patches applied). Please be aware that by default Rizin
```

Comment:
If no patches are applied then the parenthesized phrase can be removed.

Reply:
i'll keep the parentheses because the patches are still applied, they just are directly in the git repo.